File: java-mosh/A_Young_Physicist.py
T = int(input())

i_sum = 0
j_sum = 0
k_sum = 0

for i in range(T):
  i, j, k = map(int, input().split())
  i_sum += i
  j_sum += j
  k_sum += k

if i_sum == 0 and j_sum == 0 and k_sum == 0:
  print('YES')
else:
  print('NO')

File: java-mosh/A_Young_Physicist.java
import java.util.Scanner;

public class A_Young_Physicist {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    int T = scanner.nextInt();

    int iSum = 0;
    int jSum = 0;
    int kSum = 0;

    while (T > 0) {
      int i = scanner.nextInt();
      int j = scanner.nextInt();
      int k = scanner.nextInt();

      iSum += i;
      jSum += j;
      kSum += k;
      // System.out.printf("%d %d %d\n", i, j, k);
      T--;
    }

    if (iSum == 0 && jSum == 0 && kSum == 0)
      System.out.println("YES");
    else
      System.out.println("NO");
  }
}
